orders/src/main/java/nl/tudelft/wdm/group1/orders/web/OrderController.java

```diff
@@ -42,4 +40,42 @@ public Order addOrder() {
     public Order getOrder(@PathVariable(value = "id") UUID id) throws ResourceNotFoundException {
         return orderRepository.find(id);
     }
+
+    @DeleteMapping("/{id}")
+    public Order deleteOrder(@PathVariable(value = "id") UUID id) throws ResourceNotFoundException {
+        Order order = orderRepository.find(id);
+
+        producer.emitOrderDeleted(order);
+
+        return order;
+    }
+
+    @PostMapping("/{id}/items/{itemId}")
```

Comment:
Maybe we should make this `@PostMapping("/{id}/items")` with a request param for item id?

Reply:
Good point. Also for the DELETE method of that endpoint?

Reply:
I'm just assuming so :)

Reply:
Fixed!

Reply:
Actually delete is usually on a /{id} endpoint. However, since it's a bit weird anyway, I think it's fine ;).